docs/source/guidelines.rst

```diff
@@ -37,12 +37,19 @@ TODO: additional info on the various R and bioconductor URLs
 
 Hashes
 ~~~~~~
-Use `md5sum` (Linux) or `md5` (OSX) on a file to compute its md5 hash, and copy
-this into the recipe. A quick way of doing this is:
+We support either sha256 or md5 checksums to verify the integrity of the source
+package. If a checksum is provided alongside the source package (e.g. in the
+release notes or as a sister file), use that. Otherwise we prefer sha256 over md5.
+
+Use ``shasum -a 256 ...`` (or ``sha256sum``  or ``openssl sha56 ...`` etc) on a
```

Comment:
typo: `openssl sha56` => `openssl sha256`

Reply:
Maybe use `shasum -a 256 $FILE` instead of `shasum -a 256 ...` to be consistent with the `wget` example's style.
For `openssl sha256` the `...` can be omitted.